docs/book/src/cronjob-tutorial/testdata/emptymain.go

```diff
@@ -89,6 +90,40 @@ func main() {
 		os.Exit(1)
 	}
 
+	/*
+
+	Note that the Manager can restrict the namespace that all controllers will watch for resources by:
+
+	```
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Scheme: scheme,
+		Namespace: namespace,
+		MetricsBindAddress: metricsAddr}
```

Comment:
nit: To be consistent with the code block below, I'd put `}` to next line with `)`.
```
MetricsBindAddress: metricsAddr,
})
```